projects/sport-data-valley/seeds/protocols/psychological_competencies.rb

```diff
@@ -0,0 +1,36 @@
+default_protocol_duration = 1.day  # evt eerder dynamisch afbreken
```

Comment:
just 1 day, so it's a one-off questionnaire?

Reply:
Yes, it is a one-off questionnaire.